geth/api/backend.go

```diff
@@ -21,6 +21,7 @@ type StatusBackend struct {
 	accountManager common.AccountManager
 	txQueueManager common.TxQueueManager
 	jailManager    common.JailManager
+	// TODO notifer here
```

Comment:
Can you please change to `TODO(oskarth)` to conform our naming practices and help in the future refactorings?

Reply:
@divan done